packages/material-ui/src/Accordion/Accordion.js

```diff
@@ -123,33 +163,36 @@ const Accordion = React.forwardRef(function Accordion(props, ref) {
     handleChange,
   ]);
 
+  const styleProps = {
+    ...props,
+    square,
+    disabled,
+    expanded,
+  };
+
+  const classes = useUtilityClasses(styleProps);
+
   return (
-    <Paper
-      className={clsx(
-        classes.root,
-        {
-          [classes.expanded]: expanded,
-          [classes.disabled]: disabled,
-          [classes.rounded]: !square,
-        },
-        className,
-      )}
+    <AccordionRoot
+      className={clsx(classes.root, className)}
       ref={ref}
+      styleProps={styleProps}
       square={square}
       {...other}
     >
       <AccordionContext.Provider value={contextValue}>{summary}</AccordionContext.Provider>
       <TransitionComponent in={expanded} timeout="auto" {...TransitionProps}>
-        <div
+        <AccordionRegion
```

Comment:
I don't understand this change. Why not use a div?

```suggestion
        <div
```

Reply:
The accordion region had a slot, so this was to keep said slot, though it doesn't have any default styles.  Happy to remove the slot and revert these changes if desired!

Reply:
Yeah, I think that we should remove the slot. I see no valid logic for doing such. But best to wait for @mnajdova so she can validate this assumption.

Reply:
I have missed that it doesn't have any styles. Agree, we can convert to `div`, it should have better pefr that way. Good call @oliviertassinari

Reply:
Would we want to keep the class as well, or should we discard it?

Reply:
We should keep it, it would be a breaking change otherwise.